src/scss/fonts.scss

```diff
@@ -10,6 +10,28 @@ body {
     color: #32A2D2;
 }
 
+.text-charcoal-gray {
```

Comment:
These are just temporary syling, idk what looks good if someone can style it well that'd be great xd